src/styles/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  max-width: 200px;
+  min-height: 408px;
+  background-color: #fff;
```

Comment:
Consider using SCSS variables for the background color (`#fff`) to improve maintainability. Define a variable in a separate SCSS file and import it here.